3rd-party/treematch/IntConstantInitializedVector.c

```diff
@@ -24,20 +28,20 @@ void intCIV_init(int_CIVector * v, int size, int init_value)
   v->vec = malloc(sizeof(int)*size);
 }
 
-void intCIV_exit(int_CIVector * v)
+static inline void intCIV_exit(int_CIVector * v)
```

Comment:
`intCIV_exit` is renamed to `treematch_intCIV_exit` and a public function.

https://github.com/open-mpi/ompi/pull/10863/files#diff-84a819bc811ba7eb0ae298a80e8a5f11294765bcb1b4d09aa7a16a6e67d5a2adR15